Containerfile

```diff
@@ -0,0 +1,55 @@
+# This file will contain the configuration for containerized esi-leap
+
+# Use python as the parent image for esi-leap
+
+FROM python:3
+
+# Use label descriptor
+
+LABEL description="ESI-LEAP container file"
+
+# Maintainer details
+
+MAINTAINER <tzumainn@redhat.com>
+
+# The working directory for esi-leap inside the container
+
+WORKDIR /esi-leap
+
+
+# Copy the files of esi leap inside the container
+# Syntax : COPY <src> <destination>
+
+COPY . .
```

Comment:
It's generally better to be explicit regarding which files are actually needed; run_services.sh is one, there may be others.